src/specs.ts

```diff
@@ -144,14 +112,24 @@ export async function specsFromOpts(opts: Opts): Promise<BenchmarkSpec[]> {
     if (a.name !== b.name) {
       return a.name.localeCompare(b.name);
     }
-    if (a.queryString !== b.queryString) {
-      return a.queryString.localeCompare(b.queryString);
+    if (a.url.kind !== b.url.kind) {
```

Comment:
If this if for the comparison table, I would actually prefer that the specs are in order that they came on the command line.

Reply:
Done.